elastic/wazuh_cf_kibana.sh

```diff
@@ -260,7 +260,7 @@ get_plugin_url(){
       BUILD_SRC=$(pwd)
       APP_FILE=$(ls *.zip)
     else
-      plugin_url="https://aws-nist.s3-us-west-1.amazonaws.com/wazuhapp-3.11-sources.zip"
+      plugin_url="https://aws-nist.s3-us-west-1.amazonaws.com/wazuhapp-3.12-sources.zip"
```

Comment:
This resource is down.

Reply:
Fixed in fdde6b02bbceb82bdd4a4ce4be153f83c2162603